src/lib/useRoller.ts

```diff
@@ -210,30 +210,33 @@ export default function useRoller() {
       field,
       l1Txn,
       intervalTime = TEN_SECONDS,
+      pointValue,
     }: UpdateParams) => {
+
+      point = point === undefined ? await initPoint(pointValue) : point;
+
       if (l1Txn) storePendingL1Txn(l1Txn);
 
       let interval: any;
       const check = async () => {
-        const updatedPoint = await initPoint(point);
         const changedField = !points[point]
           ? 'newPoint'
-          : updatedPoint.getChangedField(points[point], field);
+          : point.getChangedField(points[point], field);
```

Comment:
I also decided to change the `checkForUpdates` api somewhat, since I noticed that after fixing the ddos-bug this interval would keep polling every ten seconds forever, since the point transfer was already present after calling `initPoint`. Callsites can still use the old behavior with `pointValue`, but most of them have access to a proper `Point` object.